src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateScriptsRule.cs

```diff
@@ -94,6 +99,16 @@ private bool CommandIsNotNeededInMSBuild(string command)
             return command.Contains("dotnet publish-iis");
         }
 
+        private static bool IsRazorPrecompilationCommand(string command)
+        {
+            return command.Contains("dotnet razor-precompile");
+        }
+
+        private static void EnableRazorCompilationOnPublish(ProjectRootElement csproj)
+        {
+            csproj.AddProperty("MvcRazorCompileOnPublish", "true");
```

Comment:
This ends up creating a separate `PropertyGroup` at the top of the csproj with the property. Do we care about trying to make it look pretty (and try and merge it in to the first `PropertyGroup`)?